client/src/app/services/theme.service.ts

```diff
@@ -9,8 +9,8 @@ export enum Theme {
 
 @Injectable()
 export class ThemeService {
-  private _currentTheme: Theme = Theme.LIGHT_THEME;
-  public readonly onChangeTheme = new BehaviorSubject<Theme>(Theme.LIGHT_THEME);
+  private _currentTheme: Theme = localStorage.getItem('theme') || Theme.LIGHT_THEME;
```

Comment:
Define a new Storage type that extends the AbstractStorage for the theme like the Token storage in the folder `client/src/app/storage`